### PR TITLE
regressed queries azure data studio notebook

### PR DESCRIPTION
**Pull request by kileri**
Showcasing how an Azure Data Studio notebook can be used to identify regressed queries in its simplest form.

### CODE REVIEW

**Comment by kileri** [comment]
@harinid and @savjani could you please review the content?

File: samples/query-store/regressed-queries/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+* Clone this repo or simply copy the notebook and the requirements.txt file into a local folder of your choice
+* Open the folder in Azure Data Studio [CTRL + K CTRL + O]
+* [Configure](https://docs.microsoft.com/en-us/sql/azure-data-studio/sql-notebooks?view=sql-server-2017#manage-packages) local Python kernel for Notebooks by switching the kernel to Python 3.
+* Have an Azure Database for PostgreSQL instance configured as needed below.
+* Update required paramaters within the notebook per blow
+* Run your notebook
+
+## Configure your database
+You need to ensure below server settings for data collection to start.
+* go into Server parameters parameters blade on Azure portal
```
Comment (savjani) [inline samples/query-store/regressed-queries/README.md]: You may want to consider using uppercase for the first letter.

File: samples/query-store/regressed-queries/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* > directional_preference is a parameter to indicate direction of interest. You should choose 1 for increase, -1 for decrease and 0 for any direction of change between baseline and current period.
+* > percent_change_threshold is a constant that can help you single out only percent changes above this value
+* > output_file_path is a local path to output your results if there is any significant changes
+* > host is in format similar to "yourdb.postgres.database.azure.com" and can be located at the overview blade of your instance in Azure portal
+* > user should be similar to "youruser@yourdb" format
+
+### I ran the notebook, now what?
+You can play with the parameters to adjust your sensitivity but the output should give you regressed query candidates, if your directional_preference is +1. Is your table stats old? Is autovacuum catching up properly to clean your bloat? Are you hitting catch querying your tables? Do you have proper indices? Is your datasize drastically changed? Has your locks been at steady state or has there been any changes?
```
Comment (savjani) [inline samples/query-store/regressed-queries/README.md]: You may want to provide some more context here like.

Once you identify the regressed queries, the next step is to identify the cause of regression,. The common causes of regression in Postgres can be 
- Out of date table statistics
- Autovacuum settings 
- Missing Indexes
- Blocking 
etc

File: samples/query-store/regressed-queries/README.md (hunk 1 above)
Comment (kileri) [reply samples/query-store/regressed-queries/README.md]: thanks @savjani ! incorporated your feedback

File: samples/query-store/regressed-queries/README.md (hunk 2 above)
Comment (kileri) [reply samples/query-store/regressed-queries/README.md]: @savjani updated the section with a little bit more detail, let me know if you think more specificity helps.

**Comment by kileri** [comment]
@rachel-msft i think with the final corrections, it is ready for your review now. thanks!

**Pull request merged**